nucliadb_node/src/bin/reader.rs

```diff
@@ -47,17 +47,19 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
     }
 
     let grpc_driver = NodeReaderGRPCDriver::from(node_reader_service);
-
-    tokio::spawn(start_grpc_service(grpc_driver));
+    let metrics_addr = SocketAddr::from(([127, 0, 0, 1], 3000));
```

Comment:
I'm pretty sure this will need to be `0.0.0.0` I think in order for it to be scrapable